lib/order.rb

```diff
@@ -0,0 +1,25 @@
+module Order
+
+  def select_dish(name,quantity)
+    raise "Sorry, dish is not available" unless dish_available?(name)
```

Comment:
Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)